common/src/main/java/com/github/khanshoaib3/minecraft_access/utils/condition/TimingKeystroke.java

```diff
@@ -9,22 +9,22 @@
  * For checking keystroke conditions that related to time,
  * like "interval/rate-limitation-on-feature-execution", "double/triple-click"
  */
-public abstract class KeystrokeTiming extends Keystroke {
+public abstract class TimingKeystroke extends Keystroke {
```

Comment:
Consider `TimedKeystroke`.
```suggestion
public abstract class TimedKeystroke extends Keystroke {
```